doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -455,22 +455,24 @@ arguments used for configuration.  Here is a table describing these emulated dev
    * - ``virtio-net``
      - Virtio network type device, parameter should be appended with the format:
        ``virtio-net,<device_type>=<name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
-       The ``mac`` address is optional, ``name`` is the name of the TAP (or MacVTap) device.
-       ``vhost`` specifies vhost backend, otherwise the VBSU backend is used.
-       ``mac_seed=<seed_string>``
-       Set a platform-unique string as a seed to generate the mac address.  Each
-       VM should have a different "seed_string".  The "seed_string" can be
-       generated by the following method where $(vm_name) contains the name of the
-       VM you are going to launch.
+       The only ``device_type`` parameter that is supported at the moment is
+       ``tap``. The ``mac`` address is optional, ``name`` is the name of the TAP
+       (or MacVTap) device. ``vhost`` specifies vhost backend, otherwise the
+       VBSU backend is used. ``mac_seed=<seed_string>`` sets a platform-unique
+       string as a seed to generate the mac address.  Each VM should have a
+       different ``seed_string``.  The ``seed_string`` can be
+       generated by the following method where ``$(vm_name)`` contains the name
+       of the VM you are going to launch.
+
+       .. code-block::
+
+          mac=$(cat /sys/class/net/e*/address)
+          seed_string=${mac:9:8}-${vm_name}
 
        .. note::
           When both ``mac`` and ``mac_seed`` are set, the latter is ignored
           and the MAC address is set to the ``mac`` value. ``mac_seed`` will
           only be used when ``mac`` is not set.
```

Comment:
Based on this note, should we tell people that ``mac`` and ``mac_seed`` are mutually exclusive arguments (you shouldn't use both)? Also, use a pipe, as follows?

``virtio-net,<device_type>=<name>[,vhost][,mac=<XX:XX:XX:XX:XX:XX> | mac_seed=<seed_string>]``

Reply:
That's what I was thinking initially. There is no error thrown when both are set though. But one is ignored. I will rephrase this and you can let me know if you like it better.

Reply:
LGTM, thanks!